simplq/src/styles/navbar.module.scss

```diff
@@ -59,6 +59,8 @@
   // Login Button
   li:last-child {
     position: fixed;
+    button {background-color: white;}
```

Comment:
The logout button at the bottom didn't have a background color, so it was not visible. Temproarily setting it to while, we can design and tweak this later:

![image](https://user-images.githubusercontent.com/11256376/97190961-60d0dc80-17cc-11eb-8621-414918cda879.png)